obspy/io/mseed/util.py

```diff
@@ -238,31 +275,110 @@ def get_timing_and_data_quality(file_or_file_object):
     info = get_record_information(file_or_file_object)
     # Keep track of the extracted information.
     quality_count = [0, 0, 0, 0, 0, 0, 0, 0]
+    activity_count = [0, 0, 0, 0, 0, 0, 0]
+    io_count = [0, 0, 0, 0, 0, 0]
```

Comment:
On Python 2 `.values()` should already return a list. In Python 3 those return iterative view objects. So this should work on both.

The way ObsPy is currently set up causes Python 2 to more or less behave like Python 3 so almost anything that works on Python 3 does also work on Python 2. If not then our tests will catch it.